### DISCUSSION TITLE
What is the difference between ojdkbuild8 and ojdkbuild8-full

### DISCUSSION THREAD

**Discussion by pheotry** [ScoopInstaller/Java#397] [General]

**Comment by se35710** [comment]
From the [Wiki](https://github.com/ojdkbuild/ojdkbuild/wiki/Motivation#windows-builds):

> ojdkbuild tries to provide [...] a "drop-in" replacement for Oracle Java 8. Historically Oracle Java 8 installer included a number of additional components that are perceived by end-users as integral parts of Java 8:
> 1. JavaFX: a full build of OpenJFX 8 (including WebKit runtime) is included with ojdkbuild MSI installer

> 2. WebStart and browser plugin: these components are not open-source, instead ojdkbuild MSI installer includes a build of [IcedTea-Web](https://icedtea.classpath.org/wiki/IcedTea-Web) - an open-source implementation of WebStart technology

> 3. Flight Recorder functionality in JVM and Mission Control tool: these features were open-sources by Oracle recently, but open source versions are not compatible with OpenJDK 8; both Flight Recorder (that is now a part of upstream OpenJDK 11+) and Mission Controls are going to be included into OpenJDK 11 ojdkbuild MSI installers

> Note, that these additional components are not based on CentOS (except IcedTea-Web, they are not available in RHEL).
